### PR TITLE
fix: global remove error message

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Give a more helpful error message if `--environment` is missing

### CODE REVIEW

File: src/cli/global/remove.rs [hunk 1]
```diff
@@ -34,13 +34,16 @@ impl HasSpecs for Args {
 }
 
 pub async fn execute(args: Args) -> miette::Result<()> {
+    let Some(env_name) = &args.environment else {
+        miette::bail!("`--environment` is required. Try `pixi global uninstall` if you want to delete the whole environment");
```
Comment (ruben-arts) [inline src/cli/global/remove.rs:38]: Could even include the `packages` names to make the command directly runnable.

**Comment by Hofer-Julian** [comment]
> Great! I am wondering if we should still support a way to shortcut
> 
> ```
> pixi global install bat
> pixi global remove bat
> ```
> 
> But maybe it's just old habits! :)

I see your point.

The reason why I like the current way more, is that it gently leads the user to the correct way, while still making it clear that `pixi global remove` is a different command, not an alias.

File: src/cli/global/remove.rs (hunk 1 above)
Comment (Hofer-Julian) [reply src/cli/global/remove.rs]: Great idea, done!

File: src/cli/global/remove.rs [hunk 2]
```diff
@@ -20,8 +20,8 @@ pub struct Args {
     packages: Vec<String>,
 
     /// Specifies the environment that the dependencies need to be removed from.
-    #[clap(short, long, required = true)]
-    environment: EnvironmentName,
+    #[clap(short, long)]
```
[the remark this answers is not in the document]
Comment (Hofer-Julian) [reply src/cli/global/remove.rs:25]: I've added a docstring at the top-level

**Pull request merged**